src/server/game/Miscellaneous/SharedDefines.h

```diff
@@ -169,7 +169,6 @@ enum FactionTemplates
     FACTION_ORC_DRAGONMAW               = 62,
     FACTION_HORDE_GENERIC               = 83,
     FACTION_ALLIANCE_GENERIC            = 84,
-    FACTION_DEMON                       = 90,
```

Comment:
no need of this changes (this and next change on this file)

Reply:
Ok. I took them out because they are no longer used in source. Better to leave them in?

Reply:
The general rule is to limit our changes to the related code.
Other changes creates "noise" and makes review harder.

Reply:
So noted - maybe I'll clean up shareddefines later. Will revert this shortly